website/common/script/content/faq.js

```diff
@@ -1,6 +1,60 @@
 import t from './translation';
 
-const NUMBER_OF_QUESTIONS = 12;
+const questionList = [
+  {
+    heading: 'overview',
+    translationIndex: 0,
+  },
+  {
+    heading: 'set-up-tasks',
+    translationIndex: 1,
+  },
+  {
+    heading: 'sample-tasks',
+    translationIndex: 2,
+  },
+  {
+    heading: 'task-color',
+    translationIndex: 3,
+  },
+  {
+    heading: 'health',
+    translationIndex: 4,
+  },
+  {
+    heading: 'party-with-friends',
+    translationIndex: 5,
+  },
+  {
+    heading: 'pets-mounts',
+    translationIndex: 6,
+  },
+  {
+    heading: 'character-classes',
+    translationIndex: 7,
+  },
+  {
+    heading: 'blue-mana-bar',
+    translationIndex: 8,
+  },
+  {
+    heading: 'monsters-quests',
+    translationIndex: 9,
+  },
+  {
+    heading: 'gems',
+    translationIndex: 10,
+  },
+  {
+    heading: 'bugs-features',
+    translationIndex: 11,
+  },
+  {
```

Comment:
is index 12 missing? or was it removed fully? 

if so - the faq.json still has the 12 entries in there

Reply:
With this structure I was able to remove it fully from the list!

I don't think it hurts to have the string tokens available in the locale files; if we ever do circle back to making another World Boss, then all we need to do is insert the entry back into this list.

Reply:
ohh ok - yeah then just keep it in there ^^